woocommerce/woocommerce
# Add new E2E tests to cover Shopper > Shop milestone

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes # [26](https://github.com/woocommerce/woocommerce-quality/milestone/26) in Woo Quality
However, the last 2 items are on hold until we decide what to do with them. I commented.

Beside all the work, I added also a workaround for activating/enabling `woocommerce_attribute_lookup_enabled` to just manually save the settings as a merchant in order for change to apply (for some reason it won't enable the setting before saving settings).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch locally
3. Execute the test `pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/product-tags-attributes.spec.js`
4. Make sure it passes without any issues



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
